Mathlib/NumberTheory/EllipticDivisibilitySequence.lean

```diff
@@ -36,8 +47,7 @@ Some examples of EDSs include
 
 ## Main statements
 
- * TODO: prove that `normEDS` satisfies `IsEllDivSequence`.
- * TODO: prove that a normalised sequence satisfying `IsEllDivSequence` can be given by `normEDS`.
+ * `isEllDivSequence_normEDS`: `normEDS` satisfies `IsEllDivSequence`.
```

Comment:
What happened to the second TODO?

Reply:
I originally forgot to add that result; it's added in [this commit](https://github.com/leanprover-community/mathlib4/pull/13057/commits/e1a47cb3867cbaf2c91be085354860ed3a06c7c1#diff-02d4668be95cd41b094c2ec5859acf1cf92c74833b9181a533bcce3a23c00961R50).